content/efcore/getting-started/samples/generating-databases-and-data-models/generating-from-code-example/Reader.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
```

Comment:
This `using` statement isn't necessary for this example because we aren't using any collections.